doc/source/changes.rst

```diff
@@ -2,6 +2,95 @@
 What's New in Workload Automation
 =================================
 
+*************
+Version 3.1.4
+*************
+
+.. warning:: This is the last release that supports Python 2. Subsequent versions
+             will be support Python 3.5+ only.
+
+New Features:
+==============
+
+Framework:
+----------
+    - A maximum and minimum version of an APK can be specified for ``ApkWorkloads``
+      instead of required a specific version.
+    - ``Artifacts`` now support the addition of directories.
+
+Workloads:
+----------
+    - Added the ``aitutu`` workload which executes the Aitutu Image Speed/Accuracy
+      and Object Speed/Accuracy tests.
+    - Added the ``uibench`` workload to run a configurable activity of the UIBench
+      workload suite.
+    - Added the ``uibenchjanktests`` workload to run an automated and instrument
+      version of the UIBench JankTests.
+    - Added the ``motionmark`` workload to run a browser graphics benchmark.
+
+Other:
+------
+    - Added ``requirements.txt`` as a reference for known working package versions.
+
+Fixes/Improvements
+==================
+
+Framework:
+----------
+    - ``JobOuputs`` now have an ``augmentation`` attribute to allow for easy listing
```

Comment:
Should the attribute name be pluralised?

Reply:
Updated.